### PR TITLE
Scunt start

### PR DESCRIPTION
**Pull request by jameskokoska**
image
image
image
image
image
image

This PR is still a WIP

TODO:
- [x] Judge panel read QR code
- [x] Category filters for missions list
- [x] Sorting filters for missions list

### CODE REVIEW

File: client/src/pages/ScuntJudgeForm/scuntTempData.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+﻿export const list = [
+  {
+    id: '1',
```
Comment (Cali0707) [inline client/src/pages/ScuntJudgeForm/scuntTempData.jsx:3]: For this list, can we switch `id` to be something else? (Anything else would be fine) - we will run into issues with this + mongoose which we are running on the backend

**Review by Cali0707 — CHANGES_REQUESTED** [review]
Very fancy! Just one small change to make this easier to work with on the backend and then this is good to go

**Review by Cali0707 — APPROVED** [review]
LGTM 🤩

**Comment by jameskokoska** [comment]
Let me get the judge qr scanning in and i'll merge after
thanks for review!

**Pull request merged**